.drone.yml

```diff
@@ -0,0 +1,26 @@
+pipeline:
+
+  test:
+    image: node:6
+    secrets:
+      - npm_config__auth
+      - npm_config_registry
+    commands:
+      - npm i
+      - npm test
+    when:
+      event: [push, pull_request, tag]
+
+  publish_to_artifactory:
+    image: node:6
+    secrets:
+      - npm_config__auth
```

Comment:
two __

Reply:
Two _ are required. The variable expected by npm is _auth not auth.  I included this in the commit message, which I'm sure you read ;)